src/Workspaces/Remote/Core/RemoteHostAssetSerialization.cs

```diff
@@ -54,8 +56,13 @@ internal static class RemoteHostAssetSerialization
             foreach (var (checksum, asset) in assetMap)
             {
                 WriteAsset(writer, serializer, context, checksum, asset, cancellationToken);
+
+                // Flush after each item as a reasonably chunk of data to want to send over the pipe.
```

Comment:
>to [](http://example.com/codeflow?start=71&length=2)

Can't parse this comment

Reply:
i can edit in followup.  It's basically saying:

> We flush after each item as that forms a reasonably sized chunk of data to want to then send over the pipe for the reader on the other side to read.  This allows the item-writing to remain entirely synchronous without any blocking on async flushing, while also ensuring that we're not buffering the entire stream of data into the pipe before it gets sent to the other side.

Reply:
Done in 00aa2df1c41.